DeluxeGrabber/manifest.json

```diff
@@ -1,10 +1,10 @@
 ﻿{
-   "Name": "Deluxe Auto-Grabber",
-   "Author": "stokastic",
-   "Version": "2.5.1",
-   "Description": "Allows use of autograbber for coops, crops, and forage",
-   "UniqueID": "stokastic.DeluxeGrabber",
-   "EntryDll": "DeluxeGrabber.dll",
-   "MinimumApiVersion": "3.0.0",
-   "UpdateKeys": [ "Nexus:2462" ]
+  "Name": "Deluxe Auto-Grabber",
+  "Author": "stokastic",
+  "Version": "2.5.3-unofficial.6-frogfrag.",
```

Comment:
Can you undo your changes to this file? Since this will be in a pull request to the official repository, it wouldn't make sense to request changing the official version to an unofficial one.